pages/profile/demographics/index.js

```diff
@@ -15,14 +17,41 @@ export default function Demographics() {
 
       <Content>
         <h1>Demographics</h1>
-        <DoughnutChart datatype="Gender" />
-        <Bar datatype="Hometown" />
-        <DoughnutChart datatype="Parental Education" />
-        <DoughnutChart datatype="Parents Born" />
+
+        <SectionTitle>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum 
+          dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+        </SectionTitle>
+
+        <Segment title="Gender" graph={<DoughnutChart datatype="Gender" />}>
+        "At vero eos et accusamus et iusto odio dignissimos ducimus q
+        </Segment>
+        <Segment title="Hometown" graph={ <Bar datatype="Hometown" />}>
+          
+        </Segment>
+        
+        <Segment title="Parents" graph={<DoughnutChart datatype="Parents Born" />}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum 
+
+        </Segment>
+        <Segment graph={
+        <>
+        <DoughnutChart datatype="Parental Education"/>
         <DoughnutChart datatype="Parents Stem" />
-        <Bar datatype="Religion" />
-        <DoughnutChart datatype="Year Born" />
-        <Bar datatype="Sexuality" />
+        </>
+        } onTop={true}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum 
+          
+        </Segment>
+
+        <Segment title="Religion" graph={<DoughnutChart datatype="Religion" />}>
```

Comment:
you might want to delete this or rebase with develop since we are not using doughnut for religion

Reply:
yeah aight got it.

Reply:
Im gonna delete this because I rebased it but it didnt change, r u sure the change of graph is on develop?

Reply:
ya I delete that part on develop but its staying for you is because you have made changes to it as well.